lambda_strip_docx/Dockerfile

```diff
@@ -27,10 +41,22 @@ RUN wget https://github.com/exiftool/exiftool/archive/refs/tags/13.38.tar.gz &&
 RUN go install github.com/pdfcpu/pdfcpu/cmd/pdfcpu@v0.11.1 && \
     cp /root/go/bin/pdfcpu /usr/local/bin/ && \
     chmod 755 /usr/local/bin/pdfcpu && \
-    rm -rf /root/go
+    rm -rf /root/go && \
+    rm -rf /usr/lib/golang
+
+# Install LibreOffice headless
+RUN wget https://download.documentfoundation.org/libreoffice/stable/25.8.2/rpm/x86_64/LibreOffice_25.8.2_Linux_x86-64_rpm.tar.gz
```

Comment:
We may need to make a note of this being a place potentially needing patches – we might be able to make Renovate care about this string, but I have no experience in how.

Reply:
will address this before I move the deployment ticket to done.